std/algebra/weierstrass/params.go

```diff
@@ -25,11 +27,525 @@ type CurveParams struct {
 func GetSecp256k1Params() CurveParams {
 	gx, _ := new(big.Int).SetString("79be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798", 16)
 	gy, _ := new(big.Int).SetString("483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8", 16)
+	g3x, _ := new(big.Int).SetString("f9308a019258c31049344f85f89d5229b531c845836f99b08601f113bce036f9", 16)
```

Comment:
Cannot we compute the table lazily first time `GetSecp256k1Params` is called? Right now one would have to check all 256*2 values if they are correct, but if we would have a simple loop
```
T <- G
for i = 1...256
    T <- T+G
end
```

then reviewer/auditor would have only check that base point and addition formulas are correct.

Or is the computation overhead too big?

Reply:
I haven't benchmarked time but it makes sense to recompute only once at init(). Feel free to push a commit if you want.